src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,69 @@
+
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+
+import com.revrobotics.CANSparkMax;
+
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.hardware.MotorController;
+import frc.robot.hardware.MotorController.MotorConfig;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInWidgets;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+public class IntakeSubsystem extends SubsystemBase implements AutoCloseable {
+
+  private CANSparkMax motor;
+  private CANSparkMax motor2;
+  public static final double OUTTAKE_SPEED = -1;
+  public static final double INTAKE_SPEED = 0.5;
+
+  /** Creates a new IntakeSubsystem. */
+  public IntakeSubsystem() {
+    motor = MotorController.constructMotor(MotorConfig.IntakeMotor1);
+    motor2 = MotorController.constructMotor(MotorConfig.IntakeMotor2);
+    motor.follow(motor2);
+ShuffleboardTab tab = Shuffleboard.getTab("Intake Status");
```

Comment:
This variable should be declared outside of the constructor, so it can be used in other methods of the class